plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlClientModule.java

```diff
@@ -43,7 +43,7 @@ public void setup(Binder binder)
         configBinder(binder).bindConfig(PostgreSqlConfig.class);
         configBinder(binder).bindConfig(JdbcStatisticsConfig.class);
         bindSessionPropertiesProvider(binder, PostgreSqlSessionProperties.class);
-        newOptionalBinder(binder, QueryBuilder.class).setBinding().to(CollationAwareQueryBuilder.class).in(Scopes.SINGLETON);
+        newOptionalBinder(binder, QueryBuilder.class).setBinding().to(PostgresAwareQueryBuilder.class).in(Scopes.SINGLETON);
```

Comment:
everything in this connector is implicitly "postgres-aware"

drop "aware"
replace "Postgres" with "PostgreSql"

Reply:
I was thinking on removing this `PostgresAwareQueryBuilder`